docs/internals/releases.md

```diff
@@ -42,9 +42,34 @@ synchronization:
 
 All publishing keys are stored as GitHub Actions in the `release` environment.
 
-## Triggering a Release
+## Versioning
+
+Because releases are frequent, we avoid specifying the version in the source code. Instead, the
+version is setup automatically by the release process itself, using sequential build numbers as the
+patch version. It might a case that a particular version is skipped (e.g., if a release partially
+failed). It is also a possibility that two releases are functionally identical (this means that no
+pull requests were merged in a week).
+
+## Release Process
 
-Releases are triggered manually, on Monday, by pushing a commit from `origin/main` to
+Releases are triggered manually, on Monday. Default release rotation is stateless and determined by
+the following one-liner:
+
+```console
+$ python3 -c "import datetime; print((sorted(['batiati', 'cb22', 'kprotty', 'matklad', 'sentientwaffle']) * 142)[datetime.datetime.utcnow().isocalendar().week:][:3])"
+['batiati', 'cb22', 'kprotty']
+```
+The middle name is the default release manager for the _current_ week. After the release, please
```

Comment:
I got a bit confused by this. Is it the middle name on the Monday that the release is meant to be done? Ie, it's @kprotty today?